packages/app/oclif.manifest.json

```diff
@@ -1445,6 +1445,78 @@
       ],
       "strict": true
     },
+    "app:import-payments-dashboard-extensions": {
```

Comment:
I thought we were looking to implement a generic `shopify app import-legacy-extensions [flags]` command which would then provide options e.g.
```
What type of extension would you like to migrate?
1. Flow
2. Checkout UI
3. Payments
```

Reply:
We've discussed this with Isaac Rolden too, but removing the existing command would likely have an impact on the current set of partners. Although these commands are probably rarely used, I'm against removing an existing command here.